create-tables.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN; CREATE TABLE public.projects (
+    "name" varchar(100) NOT NULL DEFAULT ''::character varying,
```

Comment:
It's not a big deal but I think if it's a project ID this could probably be `varchar(24)`?

Reply:
Possibly. I'm cagey about pinning the format down, as we've already had at least one format change in the past ;)

Maybe that's fine actually, depending on how much trouble it is to change a column type in postgres.

Reply:
To be fair, I don't think we really gain anything significant from changing it, so it's probably not worth it. :+1: